### ISSUE TITLE
Insert sentence shouldn't restrict the field order

### ISSUE THREAD

**Issue by darionyaphet** [vesoft-inc/nebula#1249]
```
(user@127.0.0.1) [t]> INSERT Vertex tv(fs,fi,fd,fb) VALUES 992: ("qwertyuiopasdfghjklzxcvbnmqwertyuiopasdfghjklzxcvbnm",1496245335,0.544668, true)
Execution succeeded (Time spent: 16184/16938 us)

(user@127.0.0.1) [t]> INSERT Vertex tv(fi,fb,fs,fd) VALUES 992: (1496245335,true,"qwertyuiopasdfghjklzxcvbnmqwertyuiopasdfghjklzxcvbnm",0.544668)
[ERROR (-8)]: Input field name `fi' is wrong
```

**Comment by Amber1990Zhang** [comment]
The same issue with #1215, already solved with #1219. Thanks for your suggestion.

**Comment by darionyaphet** [comment]
thanks ~ This PR is duplicated & close
